src/main/java/com/sem/btrouble/view/View.java

```diff
@@ -180,9 +192,24 @@ private void drawLives() throws SlickException {
      *            Graphics object from Slick2D
      */
     private void drawScore(Graphics graphics) throws SlickException {
+        ArrayList<Player> players = Model.getPlayers();
+        int sum = 0;
         graphics.setColor(Color.white);
-        String score = "" + GameView.getWallet().getValue();
-        graphics.drawString(score, 1090 - font.getWidth(score), 670);
+        for (int i = 0; i < players.size(); i++) {
+            String value = "Player " + (i + 1) + ": " 
+                   + Model.getWallet(Model.getPlayers().get(i)).getValue();
```

Comment:
Doesn't the Player object itself contain a wallet? 
Why is this construction used?